blog/2024-10-09-keycloakify.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Phase Two and Keycloakify, Making Keycloak Better Together
+description: Phase Two and Keycloakify deepen their relationship by becoming Sponsors and enabling Phase Two extension templates within Keycloakify.
+slug: phasetwo-keycloakify-partnership
+authors: phasetwo
+tags: [phase_two, open_source, keycloak, keycloak themes, partnership]
+---
+
+Phase Two has long been big fans of [Keycloakify](https://www.keycloakify.dev/). The work being done by [Joseph Garrone](https://github.com/garronej) is in our opinion some of the best user-focused work in the Keycloak community. Keycloakify brings the ability to theme the Keycloak frontend applications: Login, Account, Admin(coming) using modern toolsets like React or Angular and theming systems like Tailwind or Material-UI. Keycloakify allows you to build quickly and deploy themes to your Keycloak installation in a manner that the existing Keycloak toolset (ftl templates) does not.
+
+Many of Phase Two customers use Keycloakify and because we value it as an important piece of the Keycloak community, Phase Two has officially entered into a sponsorship agreement with Keycloakify.
+
+To make using our [extensions](https://github.com/p2-inc#our-extensions-) easier with Keycloakify, we've started with porting our [Magic Link](https://phasetwo.io/blog/set-up-magic-links/) [extension](https://github.com/p2-inc/keycloak-magic-link) to Keycloakify templates. At present, those templates can be found in Phase Two's forked [Keycloakify Starter](https://github.com/p2-inc/keycloakify-starter/tree/p2/magic-link-extension-templates/src/login/pages). We are actively working on adding them to the main line of templates which can be customized directly from the Keycloakify CLI.
+
+We've also joined the Keycloakify [Discord](https://discord.gg/mJdYJSdcm4) for quick and direct communication.
```

Comment:
How about adding a call to action? Something like "Please join us in building themes with Keycloakify and supporting their community!"